src/utils/module.c

```diff
@@ -71,6 +72,9 @@ gboolean bd_utils_have_kernel_module (const gchar *module_name, GError **error)
 
     path = kmod_module_get_path (mod);
     have_path = (path != NULL) && (g_strcmp0 (path, "") != 0);
+    if (!have_path) {
+      have_path = kmod_module_get_initstate (mod) == KMOD_MODULE_BUILTIN;
```

Comment:
I'd really rather see this as `builtin = kmod_module_get_initstate (mod) == KMOD_MODULE_BUILTIN;` with `gboolean builtin = FALSE;` added to the beginning of the function.